apm-sniffer/apm-sdk-plugin/kafka-plugin/src/main/java/org/apache/skywalking/apm/plugin/kafka/KafkaConsumerInterceptor.java

```diff
@@ -51,6 +51,9 @@ public void beforeMethod(EnhancedInstance objInst, Method method, Object[] allAr
     @Override
     public Object afterMethod(EnhancedInstance objInst, Method method, Object[] allArguments, Class<?>[] argumentsTypes,
         Object ret) throws Throwable {
+        if (ret == null) {
```

Comment:
When `ret == null`?

Reply:
According to `InstMethodsInter`, if the intercepted method(here is `KafkaConsumer.pollOnce()`) throws exception, the ret will be null.

Reply:
```java
Object ret = null;
        try {
            if (!result.isContinue()) {
                ret = result._ret();
            } else {
                ret = zuper.call();
            }
        } catch (Throwable t) {
            try {
                interceptor.handleMethodException(targetObject, method, allArguments, method.getParameterTypes(), t);
            } catch (Throwable t2) {
                logger.error(t2, "class[{}] handle method[{}] exception failure", obj.getClass(), method.getName());
            }
            throw t;
        } finally {
            try {
                ret = interceptor.afterMethod(targetObject, method, allArguments, method.getParameterTypes(), ret);
            } catch (Throwable t) {
                logger.error(t, "class[{}] after method[{}] intercept failure", obj.getClass(), method.getName());
            }
        }
```

Reply:
Look good to me, could you add comments about this at `if (ret == null) {` and  ` if (ContextManager.isActive())` places?

Reply:
Updated.

Reply:
Look like `finagle` doesn't have the log? I assume they are the same cases?

Reply:
Comments were Added to finagle too!